doc: Updated documentation for CS2 release (#17416)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs2.rst

  1. Open `nRF Connect for Desktop`_.
  #. Open the Toolchain Manager application in nRF Connect for Desktop.
  #. Click the button with the arrow pointing down next to the installed |NCS| version (v2.7.0) to expand the drop-down menu with options.
  #. In the drop-down menu, click :guilabel:`Update toolchain`.
  #. In the same drop-down menu, click :guilabel:`Update SDK`.

nrfutil device
  ``nrfutil device`` has been updated to version 2.5.4.

  Install the nRF Util ``device`` command version 2.5.4 as follows::

     nrfutil install device=2.5.4 --force

  For more information, consult the `nRF Util`_ documentation.

nrfutil trace
  ``nrfutil trace`` has been updated to version 2.11.0.

  Install the nRF Util ``trace`` command version 2.11.0 as follows::

     nrfutil install trace=2.11.0 --force

  For more information, consult the `nRF Util`_ documentation.

nRF54H20 BICR
  The nRF54H20 BICR has been updated.

  To update the BICR of your development kit while in Root of Trust, do the following:

  1. Download the `BICR new binary file`_.
  #. Connect the nRF54H20 DK to your computer using the **DEBUGGER** port on the DK.

     .. note::

>         On MacOS, connecting the DK might cause a popup containing the message ``Disk Not Ejected Properly`` to repeatedly appear on screen.

Reviewer: ```suggestion
        On MacOS, connecting the DK may repeatedly trigger a popup displaying the message ``Disk Not Ejected Properly``.
```

  1. Download the `nRF54H20 firmware bundle v0.6.4`_.

     .. note::
        On MacOS, ensure that the ZIP file is not unpacked automatically upon download.

  #. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, and then run nRF Util to program the :file:`nordic_top.suit` using the following command::

        nrfutil device x-suit-dfu --firmware nordic_top.suit --serial-number <serialnumber>

New SEGGER J-Link version
  A new version of SEGGER J-Link is supported: SEGGER `J-Link version 7.94i`_.

  .. note::
    On Windows, to update to new the J-link version, including the USB Driver for J-Link, you must manually install J-Link v7.94i from the command line using the ``-InstUSBDriver=1`` parameter, updating the installation previously run by the `nRF Command Line Tools`_:

    1. Navigate to the download location of the J-Link executable and run one of the following commands:

        * From the Command Prompt::

          JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1

        * From PowerShell::

          .\JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1

>     #. In the :guilabel:`Choose optional components` window, select again :guilabel:`update existing installation`.

Reviewer 2: Why is this "again"? I don't see the option selected in any other step.
